models/test_suite/kade/conventions/convention.ka

```diff
@@ -4,6 +4,6 @@
 %init: 10 A()
 %init: 10 B()
 
-A(),A() -> A() @1
-A(),A(),A(),B(),B() -> A(),A(),B(),B() @1
-B(),B(),B() -> B(),B() @1 
+A(),A() -> A(),. @ 1
+A(),A(),A(),B(),B(),.,. -> A(),A(),.,.,.,B(),B() @ 1
```

Comment:
This translation is accurate but is it what the user meant or did she wanted to say `A(),A(),A(),B(),B() -> A(),A(),.,B(),B() @ 1`? (and if so should we keep the mistake or correct it?)

Reply:
Yes I meant this: 
A(),A(),A(),B(),B() -> A(),A(),.,B(),B() @ 1

And I would prefer to keep it as it is. 

Thank you Pirbo.